### ISSUE TITLE
训练时loss=0，Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.是什么原因

### ISSUE THREAD

**Issue by jy-101361-1810897** [dwzhu-pku/PoSE#16]
bash script/run_train_baichuan.sh 64 yarn

factor=$1
rope_type=$2

debug_mode="-m debugpy --listen 127.0.0.1:6679 --wait-for-client"
# python -m torch.distributed.run --nproc_per_node=1 ${debug_mode} src/train_baichuan.py \
deepspeed src/train_baichuan.py \
    --model_name_or_path ./baichuan2-7b-base \
    --train_data_path ./data/pile/train_00_long_10w.jsonl \
    --valid_data_path ./data/pile/val_long.jsonl \
    --test_data_path ./data/pile/test_pg19.jsonl \
    --output_dir ./skipos/results/baichuan2/4k-$((factor*4))k-${rope_type} \
    --max_steps 1000 \
    --model_max_position_embeddings 4096 \
    --rope_scaling_type ${rope_type} \
    --rope_scaling_factor $factor \
    --inference_length 16384 \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps 2 \
    --do_train True \
    --do_eval True \
    --do_predict True \
    --evaluation_strategy "steps" \
    --eval_steps 50 \
    --save_strategy "steps" \
    --save_steps 500 \
    --warmup_steps 0 \
    --learning_rate 2e-5 \
    --logging_steps 10 \
    --report_to "tensorboard" \
    --gradient_checkpointing True \
    --fp16 True \
    --deepspeed src/configs/deepspeed_config.json \
